index.html

```diff
@@ -9,30 +9,38 @@
     <title>Portfolio Project</title>
 </head>
 <body>
+    <header>
     <nav class="nav-bar">
+        <a href="#">
         <h1 class="title">Welcome!</h1>
+        </a>
         <a class="icon">
             <i class="fa fa-bars"></i>
         </a>
     </nav>
-    <section class="headline">
+    </header>
+    <section class="main-container">
+    <div class="headline">
         <div class="intro-txt">
-            <h2 class="wave-txt">Hey there. I'm Lucas</h2>
-            <h2 class="desc-txt">I'm a software developer</h2>
+            <h1 class="wave-txt">Hey there. I'm Lucas</h1>
+            <h1 class="desc-txt">I'm a software developer</h1>
         </div>
         <div class="main-headline-txt">
             <p>
-            I am begginer student at Microverse, currently learning how to implement flexbox properties in CSS. This is
-            my first project using flexbox! Below are the social media links to contact me
+                I can help you build a product, feature or website Look through some of my work and experience! 
+                If you like what you see and have a project you need coded, don't hestiate to contact me.
             </p>
         </div>
         <div class="media-imgs">
-            <img src="imgs/twitter.svg" alt="Twitter img">
-            <img src="imgs/Linkedin-icon.svg" alt="Linkedin img">
-            <img src="imgs/medium.svg" alt="Medium img">
-            <img src="imgs/github.svg" alt="Github img">
-            <img src="imgs/angellist.svg" alt="Angellist img">
+            <ul>
+            <li><img src="imgs/twitter.svg" alt="Twitter img"></li>
+            <li><img src="imgs/Linkedin-icon.svg" alt="Linkedin img"></li>
+            <li><img src="imgs/medium.svg" alt="Medium img"></li>
+            <li><img src="imgs/github.svg" alt="Github img"></li>
+            <li><img src="imgs/angellist.svg" alt="Angellist img"></li>
+            </ul>

```

Comment:
[OPTIONAL]

- [ ] I suggest that you fix the indentation here, as it will make your code organized and easy to read.